docs/madr/decisions/070-resource-identifier.md

```diff
@@ -0,0 +1,99 @@
+# Resource Identifier
+
+* Status: accepted
+
+## Context and Problem Statement
```

Comment:
There's prior work around this from @Automaat and @lukidzi we should reference it here.

Lukasz made a MADR about cluster names
Automaat made a MADR about internal envoy entities

Reply:
It's also fine mentioning the PR and the few issues that mention the need for this

Reply:
Referenced both MADRs in the context.